sympy/utilities/codegen.py

```diff
@@ -117,45 +117,37 @@
 
 class Routine(object):
     """Generic description of evaluation routine for set of expressions.
-
```

Comment:
You should leave this new line be. (first line of a docstring should be a one sentence summary)